tgui/packages/tgui/interfaces/portagrav.tsx

```diff
@@ -0,0 +1,134 @@
+import { BooleanLike } from 'common/react';
+
+import { useBackend } from '../backend';
+import { Box, Button, Icon, Section, Stack } from '../components';
+import { Window } from '../layouts';
+
+type Data = {
+  percentage: number;
+  on: BooleanLike;
+  range: number;
+  maxrange: number;
+  gravity: number;
+  wiremode: BooleanLike;
+  draw: number;
+};
+
+export const portagrav = (props) => {
```

Comment:
why is this lower case? react components, and their containing files, are pascal case